### ISSUE TITLE
(ZIP) (PS1) [Package Request]: ContextMenuForWindows11 

### ISSUE THREAD

**Issue by ppvnf** [microsoft/winget-pkgs#80954]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [ ] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: https://github.com/ikas-mc
Publisher Url: https://github.com/ikas-mc/ContextMenuForWindows11
Package Name: ContextMenuForWindows11
Description: Add Custom Context Menu For Windows11
Package Version: 3.1.0.0
Installer URL: https://github.com/ikas-mc/ContextMenuForWindows11/releases

Besides being inside a .zip file the installer seems to be a .ps1 file. I know Zip support will come soon, but is Winget able to to execute installation scripts?

**Comment by Trenly** [comment]
> is Winget able to to execute installation scripts?

At this time, no. There are security risks associated with downloading and running installation scripts with little ability to validate them, especially since such scripts often rely on reaching out to external sources to download the binary executables.

**Comment by Trenly** [comment]
I don't know that there has been any discussion around this. I know #21877 exists here in pkgs, but I couldn't find anything over at CLI either

**Comment by ppvnf** [comment]
Maybe this https://github.com/microsoft/winget-cli-restsource/issues/116?

**Comment by Trenly** [comment]
> Maybe this [microsoft/winget-cli-restsource#116](https://github.com/microsoft/winget-cli-restsource/issues/116)?

No, that is for .exe files that aren’t packaged as an installer. If the .ps1 file references a .exe, it may be possible to add the .exe directly

**Comment by ppvnf** [comment]
latest version is available as Msixbundle
https://github.com/ikas-mc/ContextMenuForWindows11/releases/download/3.8.0.0/7061touchwp.CustomContextMenu_3.8.0.0_neutral_._m9vp3t2f55f5t.Msixbundle

**Comment by ppvnf** [comment]
available on ms store
